ROADMAP.md

```diff
@@ -0,0 +1,102 @@
+# ROADMAP for IPFS in Web Browsers
+
+> This page provides the community with a high level overview of our current priorities and ongoing endevours.
+
+### Table of Contents
+
+* [ROADMAP](#roadmap)
+    * [Goals](#goals)
+    * [Prerequisites](#prerequisites)
+    * [Long Game](#long-game)
+* [Quarterly Objectives and Key Results](#quarterly-objectives-and-key-results)
+    * [Q2 2018](#q2-2018)
+    * [Q1 2018](#q1-2018)
+        * [We have a shared understanding of how IPFS <em>should work</em> in web browsers](#we-have-a-shared-understanding-of-how-ipfs-should-work-in-web-browsers)
+        * [Browser developers understand the requirements of the decentralized web](#browser-developers-understand-the-requirements-of-the-decentralized-web)
+        * [Move towards IPFS working seamlessly in browsers](#move-towards-ipfs-working-seamlessly-in-browsers)
+        * [Provide a nice desktop experience for casual users of IPFS](#provide-a-nice-desktop-experience-for-casual-users-of-ipfs)
+
+# ROADMAP
+
+### Goals
+- **Native support in the Browsers:** protocol handlers for decentralized web protocols, especially IPFS and IPLD, bundled natively in web browsers.
+- **Dapps are just.. apps:** creating web applications with IPFS is a viable alternative to HTTP-based stack.
+
+### Prerequisites
+- *DONE:* Implement [js-ipfs](https://github.com/ipfs/js-ipfs)
+- *DONE:* Make [js-ipfs](https://github.com/ipfs/js-ipfs) and [go-ipfs](https://github.com/ipfs/go-ipfs) interoperate seamlessly
+- *DONE:* Create browser extension ([ipfs-companion](https://github.com/ipfs-shipyard/ipfs-companion)) that detects IPFS resources and redirects requests to local gateway.
+
+### Long Game
+- *IN PROGRESS:* Improve extension so it handles distributed protocols natively without need for third party software.
+- *IN PROGRESS:* Define the APIs, address schemes, security models, etc.
+- *IN PROGRESS:* Make the case for IPFS & Decentralized Web Protocols in Web Browsers with Use Cases, Sample Applications, Demos and Delightful Documentation
+- *IN PROGRESS:* Work with browser vendors on adding missing APIs and solving discovered problems.
+
+
+# Quarterly Objectives and Key Results
```

Comment:
I'd drop the Q1 and Q2 2018 headers and just collapse this into a single

`Objectives and Key Results - Q1 2018` as a h2

The content is all good.

Reply:
Question is: are we going to replace this section every quarter or  do we want to keep old records and grow it every quarter?

Reply:
I lean slightly towards "replace" as this isn't a changelog, but more a statement of what we're focusing on currently. Might be cool to be able to read the the full list somewhere in a couple of years time, but that could be a separate doc.

Reply:
Ok, sounds good. I removed Q2 items for now, but kept header names.
We will revisit this when OKRs for Q2 are in.